modules/manage/pages/manage-nodes/add-node-and-rebalance.adoc

```diff
@@ -116,9 +116,12 @@ The pane expands to provide additional information on the progress of data-trans
 image::manage-nodes/rebalanceOpenedDataTab.png[,430,align=middle]
 +
 The dialog provides the average times required in the move of each vBucket.
-A set of averages is provided for each bucket on the cluster.
-For an explanation of the _stages_ &#8212; *Move*, *Backfill*, *Takeover*, and *Persistence* &#8212; see xref:learn:clusters-and-availability/rebalance.adoc#data-service-rebalance-stages[Data-Service Rebalance Stages].
-Note that the dialog may also include progress information on data-compression.
+A set of constantly updated figures is provided for each bucket on the cluster.
+For a detailed explanation of the _stages_ &#8212; *Move*, *Backfill*, *Takeover*, and *Persistence* &#8212; see xref:learn:clusters-and-availability/rebalance.adoc#data-service-rebalance-stages[Data-Service Rebalance Stages].
+Note that the figure for *Persistence* relates only to _Couchbase_ buckets, and therefore remains at zero wherever the display is providing status on an _Ephemeral_ bucket.
+Note also that the *Takeover* figure is specific only to _active_ vBuckets: therefore it remains at zero when only _replica_ vBuckets are being moved.
++
+The dialog may also include progress information on data-compression.
```

Comment:
I did a fetch and rebase before I committed Amarantha's required edits. These changes were merged into the pull request then.

Reply:
I agree it is confusing. I guess this will occur when a PR stays open for a long time, with edits being pushed at intervals. For comparison, Gerrit is pain to set up and slightly more difficult to use than GitHub reviews, but it does avoid this issue. I'll do some reading and see if I can find out what the best practise is.

Reply:
Usually when I rebase, those commits don't show up in the PR because my commits are played back on top those commits in the history. These things work themselves out when I force-push new changes to my fork. If you were doing work directly on the PR, perhaps this is how it looks in the history 🤔

Reply:
If you look at the commit from this pull request fb6d5e3 it only contains the changes that were made as part of this commit — not any of the others.

Reply:
Must be GitHub just displaying one of the many underlying git paradigms that I don't understand.